client/src/GMRoute/GMInGame/Tabs/ActionsTab/NightActions.tsx

```diff
@@ -40,17 +39,11 @@ const gameActionsByNight = {
   otherNight: [] as Action[],
 };
 
-interface NightPlayerListProps {
-  firstNight: boolean;
-  endNightCallback?: () => void;
-}
-export function NightPlayerList({
-  firstNight,
-  endNightCallback = () => {},
-}: NightPlayerListProps) {
+interface NightActionsProps {}
+export function NightActions(_props: NightActionsProps) {
   const { game } = useDefiniteGame();
-  const [_, setIsHiddenView] = useIsHiddenView();
 
+  const firstNight = game.gameStatus === "Setup";
```

Comment:
Should we still be using status to determine first day when we've now got game.time?

Reply:
need to go a step further still on status.